lib/vyasa_web/live/media_live/media_bridge.ex

```diff
@@ -524,7 +524,7 @@ defmodule VyasaWeb.MediaLive.MediaBridge do
       }
     >
       <%= if not @isReady do %>
-        <.spinner />
+        <.icon name="custom-spinner-bars-scale-middle" class="w-6 h-6" />
```

Comment:
# Custom SVG Icons

But not sure how to scale this properly

![image](https://github.com/user-attachments/assets/7626b204-d85c-4067-a3f9-656d801708ad)

Reply:
note to self: when vendoring SVG icons, ensure that it's dimensions are NOT defined so that they can be defined where they are used.